src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1195,7 +1195,9 @@ public Collection<Object> createComponents(
                 resolver,
                 xContentRegistry,
                 threadPool,
-                dlsFlsBaseContext
+                dlsFlsBaseContext,
+                adminDns,
+                resourcePluginInfo != null ? resourcePluginInfo.getResourceIndices() : null
```

Comment:
resourcePluginInfo will never be null since we instantiate it at class-level.

Reply:
Changed.